.github/workflows/sngls-search-workflow.yml

```diff
@@ -0,0 +1,58 @@
+name: run small singles-included search using pegasus + condor
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: install condor
+      run: |
+        wget -qO - https://research.cs.wisc.edu/htcondor/ubuntu/HTCondor-Release.gpg.key | sudo apt-key add -
+        echo "deb http://research.cs.wisc.edu/htcondor/ubuntu/8.9/focal focal contrib" | sudo tee -a /etc/apt/sources.list
+        echo "deb-src http://research.cs.wisc.edu/htcondor/ubuntu/8.9/focal focal contrib" | sudo tee -a /etc/apt/sources.list
+        sudo apt-get update
+        sudo apt-get install minihtcondor
+        sudo systemctl start condor
+        sudo systemctl enable condor
+    - name: install pegasus
+      run: |
+        wget https://download.pegasus.isi.edu/pegasus/ubuntu/dists/bionic/main/binary-amd64/pegasus_5.0.1-1+ubuntu18_amd64.deb
+        sudo apt install ./pegasus_5.0.1-1+ubuntu18_amd64.deb
+    - run: sudo apt-get install *fftw3* intel-mkl*
+    - name: Install pycbc
+      run: |
+        python -m pip install --upgrade 'pip<22.0' setuptools
+        pip install -r requirements.txt
+        pip install .
+    - name: retrieving frame data
+      run: bash -e examples/search/get.sh
+    - name: generating template bank
+      run: bash -e examples/search/bank.sh
+    - name: generating statistic files
+      run: bash -e examples/search/stats.sh
+    - name: running workflow
+      run: |
+        cp examples/singles_search/*.ini ./
+        bash -e examples/singles_search/gen.sh
+        cp *.gwf output/
+        cd output
+        bash -e ../examples/search/submit.sh
+        python ../examples/search/check_job.py
+        find submitdir/work/ -type f -name '*.tar.gz' -delete
+    - name: store log files
+      if: always()
+      uses: actions/upload-artifact@v2
+      with:
+        name: logs
+        path: output/submitdir/work
+    - name: store result page
+      uses: actions/upload-artifact@v2
+      with:
+        name: results
+        path: html
```

Comment:
Currently not uploading any result page - once I've worked it out, this will be a separate PR, so this is just filler for now